packages/lesswrong/lib/collections/posts/callbacks.js

```diff
@@ -155,3 +155,12 @@ async function LWPostsNewUpvoteOwnPost(post) {
 addCallback('posts.new.after', LWPostsNewUpvoteOwnPost);
 
 addEditableCallbacks({collection: Posts, options: makeEditableOptions})
+
+function PostsNewUserApprovedStatus (post) {
+  const postAuthor = Users.findOne(post.userId);
+  if (!postAuthor.reviewedByUserId) {
+    return {...post, unreviewedUser: true}
```

Comment:
This seems to be making it so that all posts need to be reviewed before they can appear, even posts by admins and by users with very high karma? That seems incorrect.